### PR TITLE
Include a single test runner and disable coverage on Travis

### LINKED ISSUE #57: Use templating to only include the desired test runner
http://jinja.pocoo.org/docs/dev/templates/#equalto

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/6708296-use-templating-to-only-include-the-desired-test-runner?utm_campaign=plugin&utm_content=tracker%2F1487757&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F1487757&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by jacebrowning**
Fixes #57 and #71.

### CODE REVIEW

**Comment by jacebrowning** [comment]
@joshfriend I'm addressing #57 and #71 here. You can test this locally via:

    TEST_RUNNER=nose make clean ci
    TEST_RUNNER=pytest make clean ci

and view the generated `Makefile` in each case. What do you think?

File: {{cookiecutter.project_name}}/Makefile [hunk 1]
```diff
@@ -99,8 +95,10 @@ depends: depends-ci depends-dev
 .PHONY: depends-ci
 depends-ci: env Makefile $(DEPENDS_CI)
 $(DEPENDS_CI): Makefile
-	$(PIP) install --upgrade pep8 pep257 pylint $(TEST_RUNNER) coverage
-	touch $(DEPENDS_CI)  # flag to indicate dependencies are installed
+	$(PIP) install --upgrade pep8 pep257 pylint coverage
+{% if cookiecutter.test_runner == "nose" %}	$(PIP) install --upgrade nose
```
Comment (joshfriend) [inline {{cookiecutter.project_name}}/Makefile]: You can clean up the generated makefile (and the template) a little by using [whitespace control](http://jinja.pocoo.org/docs/dev/templates/#whitespace-control) and [jinja variables](http://jinja.pocoo.org/docs/dev/templates/#assignments):

```patch
diff --git a/{{cookiecutter.project_name}}/Makefile b/{{cookiecutter.project_name}}/Makefile
index 34069fc..0ea2c24 100644
--- a/{{cookiecutter.project_name}}/Makefile
+++ b/{{cookiecutter.project_name}}/Makefile
@@ -95,10 +95,9 @@ depends: depends-ci depends-dev
 .PHONY: depends-ci
 depends-ci: env Makefile $(DEPENDS_CI)
 $(DEPENDS_CI): Makefile
-       $(PIP) install --upgrade pep8 pep257 pylint coverage
-{% if cookiecutter.test_runner == "nose" %}    $(PIP) install --upgrade nose
-{% elif cookiecutter.test_runner == "pytest" %}        $(PIP) install --upgrade pytest
-{% endif %}    touch $(DEPENDS_CI)  # flag to indicate dependencies are installed
+       {% set dev_packages = ['pep8', 'pep257', 'pylint', 'coverage', cookiecutter.test_runner] -%}
+       $(PIP) install --upgrade {{ ' '.join(dev_packages) }}
+       touch $(DEPENDS_CI)  # flag to indicate dependencies are installed

 .PHONY: depends-dev
 depends-dev: env Makefile $(DEPENDS_DEV)
```

**Comment by joshfriend** [comment]
Works fine for me :+1:

File: {{cookiecutter.project_name}}/Makefile (hunk 1 above)
Comment (jacebrowning) [inline {{cookiecutter.project_name}}/Makefile]: Thanks!

**Pull request merged**